src/dappsExplorer/searchFilter/DAppsExplorerScreenSearchFilter.test.tsx

```diff
@@ -0,0 +1,640 @@
+import { fireEvent, render, within } from '@testing-library/react-native'
```

Comment:
is this file all copy paste, or are there key changes that should be reviewed?

Reply:
Only this section is new: https://github.com/valora-inc/wallet/blob/193a9bc20bd2096c4b64d54324e91db001d73ae5/src/dappsExplorer/searchFilter/DAppsExplorerScreenSearchFilter.test.tsx#L564-L640.